launcher/minecraft/Library.cpp

```diff
@@ -126,6 +127,10 @@ QList<NetAction::Ptr> Library::getDownloads(
         // Don't add a time limit for the libraries cache entry validity
         options |= Net::Download::Option::MakeEternal;
 
+        if (useBMCLAPI) {
+            url.replace(BuildConfig.LIBRARY_BASE, BuildConfig.BMCLAPI_LIBRARY_BASE);
```

Comment:
Does BCML also cover older versions that are hosted on `launcher.mojang.org`?

See https://github.com/PrismLauncher/meta-launcher/blob/master/net.minecraft/1.4.3.json for example